apps/python-sdk/firecrawl/v2/types.py

```
@@ -775,12 +775,63 @@
+class VectorSearchRequest(BaseModel):
+    """Request for vector search operations."""
+    query: str
+    limit: Optional[int] = None
+    offset: Optional[int] = None
+    threshold: Optional[float] = None
+    include_content: Optional[bool] = None
+    filters: Optional[VectorSearchFilters] = None
+    origin: Optional[str] = None
+    integration: Optional[str] = None
+
+class VectorSearchResult(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)